packages/framework/build.xml

```diff
@@ -403,4 +401,724 @@
         </delete>
     </target>
 
+    <target name="build-dev" depends="generate-build-version,clean,redis-check,clean-redis-old,composer-dev,npm,timezones-check,dirs-create,test-dirs-create,assets,db-migrations,create-domains-data,generate-friendly-urls,replace-domains-urls,product-search-recreate-structure,product-search-export-products,grunt,error-pages-generate,tests-acceptance-build,checks-diff" description="Builds application for development preserving your DB and runs checks on changed files."/>
+    <target name="build-dev-quick" depends="generate-build-version,clean,redis-check,clean-redis-old,composer-dev,npm,dirs-create,test-dirs-create,assets,db-migrations,create-domains-data,generate-friendly-urls,replace-domains-urls,grunt,tests-acceptance-build" description="Builds application for development preserving your DB while skipping nonessential steps."/>
+    <target name="build-demo-dev" depends="generate-build-version,wipe-excluding-logs,redis-check,clean-redis-old,composer-dev,npm,timezones-check,dirs-create,test-dirs-create,assets,db-demo,product-search-recreate-structure,product-search-export-products,grunt,error-pages-generate,tests-acceptance-build,checks-diff" description="Builds application for development with clean demo DB and runs checks on changed files."/>
+    <target name="build-demo-dev-quick" depends="generate-build-version,wipe-excluding-logs,redis-check,clean-redis-old,composer-dev,npm,dirs-create,test-dirs-create,assets,db-demo,product-search-recreate-structure,product-search-export-products,grunt,tests-acceptance-build" description="Builds application for development with clean demo DB while skipping nonessential steps."/>
+    <target name="build-demo-ci" depends="generate-build-version,wipe-excluding-logs,redis-check,clean-redis-old,timezones-check,dirs-create,test-dirs-create,db-demo,product-search-recreate-structure,product-search-export-products,grunt,error-pages-generate,checks-ci" description="Builds application for development with clean demo DB and runs CI checks."/>
```

Comment:
I was thinking about removing the `˙*-ci` targets completely, as everyone can have a slightly different QA pipeline. I didn't get to doing it because all the extra work fixing up all scripts and docs and explaining how to replicate that work in the project repository via upgrade notes...

If you don't mind, I would rather avoid tweaking those two targets more than necessary.

Reply:
it would solve #565, but i am OK with removing *-ci